app/styles/themes/gothamist/_theme.overrides.scss

```diff
@@ -650,3 +650,30 @@ blockquote {
 .c-block--horizontal .c-block__title {
   @include typeface(header, 8);
 }
+
+// icons
+
+.u-icon--xs {
+  width: 16px;
+  height: 16px;
+}
+
+.u-icon--s {
+  width: 20px;
+  height: 20px;
+}
+
+.u-icon--m {
+  width: 30px;
+  height: 30px;
+}
+
+.u-icon--l {
+  width: 40px;
+  height: 40px;
+}
+
+.u-icon--xl {
+  width: 50px;
+  height: 50px;
+}
```

Comment:
It feels redundant to have these class-based width/height attributes in addition to the hardcoded width/height attributes above. Many of the values overlap. Wondering if there is a way to consolidate these to avoid duplication?

Reply:
to follow up -- do these two directives from Matt's notes contradict each other? -- 
- size: use css classes
- don’t need any tokens, just hardcode the icon sizes in their css mod classes

Reply:
Thanks, Brad :) For "size: use css classes", Matt meant to use css to size the icons as opposed to using a size property on the component. The Gothamist sizes were a bit different than the whitelabel... maybe I should ask design if Gothamist can conform or is we need those overrides.